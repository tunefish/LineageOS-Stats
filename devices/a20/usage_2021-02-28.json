{
  "17.1": 2416,
  "none": 24
}