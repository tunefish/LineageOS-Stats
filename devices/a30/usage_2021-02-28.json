{
  "17.1": 29,
  "18.0": 670,
  "18.1": 255,
  "none": 4
}