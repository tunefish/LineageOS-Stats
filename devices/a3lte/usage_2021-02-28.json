{
  "15.1": 623,
  "16.0": 218,
  "17.1": 705
}