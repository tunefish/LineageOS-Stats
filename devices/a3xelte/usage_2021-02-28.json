{
  "14.1": 3,
  "15.1": 119,
  "16.0": 486,
  "17.1": 3807,
  "18.0": 3,
  "18.1": 14,
  "52.8": 1,
  "none": 2
}