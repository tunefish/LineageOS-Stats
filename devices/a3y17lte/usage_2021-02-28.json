{
  "14.1": 5553,
  "15.1": 361,
  "16.0": 1070,
  "17.1": 1062,
  "18.0": 1
}