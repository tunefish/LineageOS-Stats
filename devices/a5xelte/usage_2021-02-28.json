{
  "15.1": 56,
  "16.0": 555,
  "17.1": 4513,
  "18.0": 12,
  "18.1": 3,
  "none": 1
}