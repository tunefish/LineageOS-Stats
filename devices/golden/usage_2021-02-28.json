{
  "13.0": 1100,
  "14.1": 3610,
  "15.1": 4,
  "17.1": 5,
  "none": 1
}